pyiron/vasp/base.py

```diff
@@ -1420,6 +1420,33 @@ def restart_for_band_structure_calculations(self, job_name=None):
             self_consistent_calc=None
         )
 
+    def _get_icharg_value(self, icharg=None, self_consistent_calc=None):
```

Comment:
I like this function but I think it can be made public.

Reply:
okay 😄